src/main/java/de/tum/cit/aet/artemis/programming/domain/ProgrammingExerciseBuildConfig.java

```diff
@@ -49,7 +49,13 @@ public class ProgrammingExerciseBuildConfig extends DomainObject {
     private boolean checkoutSolutionRepository = false;
 
     @Column(name = "checkout_path")
-    private String checkoutPath;
+    private String testCheckoutPath;
```

Comment:
NIT: I think we could also update the column name, but I don't have a strong opinion on that

Reply:
Change the entry in the database would make it harder to test on TS since deployed this PR would break the TS for other PRs